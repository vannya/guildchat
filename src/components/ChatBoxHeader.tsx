import React from 'react';
export interface IChatBoxHeaderProps {
	userName: string;
}

export const ChatBoxHeader = ({
	userName
}: IChatBoxHeaderProps): JSX.Element => {
	return <h2 className='chatbox-header'>Chatting with {userName}</h2>;
};
