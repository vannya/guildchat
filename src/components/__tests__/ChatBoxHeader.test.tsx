import { cleanup, render } from '@testing-library/react';
import React from 'react';
import { ChatBoxHeader, IChatBoxHeaderProps } from '../ChatBoxHeader';

describe('ChatBoxHeader', () => {
	const getProps = (): IChatBoxHeaderProps => {
		return {
			userName: 'Test Name'
		};
	};

	it('should match snapshot', () => {
		const props = getProps();
		const { container } = render(<ChatBoxHeader {...props} />);
		expect(container).toMatchSnapshot();
		cleanup();
	});
});
