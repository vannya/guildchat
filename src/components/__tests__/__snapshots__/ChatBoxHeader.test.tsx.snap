// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatBoxHeader should match snapshot 1`] = `
<div>
  <h2
    class="chatbox-header"
  >
    Chatting with 
    Test Name
  </h2>
</div>
`;
